app.js

```diff
@@ -0,0 +1,19 @@
+import express from 'express';
+import routes from './routes/routes.js';
+import { checkUserId } from './middleware/checkUserId.js';
+import { errorHandler } from './middleware/errorHandler.js';
+
+const app = express();
+const PORT = 3000;
+
+app.use(express.json());
+
+app.use(checkUserId);
```

Comment:
this middleware is executed before each route but for some routes, for example, /register, /products this check is not necessary. Could you apply the middleware that checks user id only to specific routes please(check requirements for previous homework)?